firebase-firestore/src/main/java/com/google/firebase/firestore/local/MemoryLruReferenceDelegate.java

```diff
@@ -170,4 +179,18 @@ private boolean isPinned(DocumentKey key, long upperBound) {
     Long sequenceNumber = orphanedSequenceNumbers.get(key);
     return sequenceNumber != null && sequenceNumber > upperBound;
   }
+
+  @Override
+  public long getByteSize() {
+    // Note that this method is only used for testing because this delegate is only
+    // used for testing. The algorithm here (loop through everything, serialize it
+    // and count bytes) is inefficient and inexact, but won't run in production.
+    long count = 0;
```

Comment:
This will change though, won't it? I thought we wanted to run LRU instead of EagerGC for memory persistence as well.